ignore_words.txt

```diff
@@ -4,3 +4,6 @@ WAN
 filname
```

Comment:
Should "filename" be ignored. Should it be "filename" instead?

Reply:
Hi @PipKat, see other comment on this file.